Tutorial translation in english (#1)
InseeFrLab/funcamp-r-grimoire: inst/learnr/Spellbook_IGoR/chapter1/chapter1.Rmd

Unlike other software like SAS, R is a **free software**: its license is free and allows each user to access, modify or redistribute its **source code**. The source code is the equivalent of the manual for making the software. It is also cross-platform: this means that it can be run on several operating systems (Windows, Mac, Linux ...)

## Des Quiz tu triompheras
## Quizzes you will defeat

Dans l'aventure d'icaRius, vous serez parfois amené(e) à répondre à des quiz pour aider le personnage à poursuivre sa quête. En voici un tout premier, histoire de se familiariser avec le format de l'exercice. À vous donc de répondre directement en cochant votre choix puis en appuyant sur le bouton qui s'intitule `Submit answer` (*envoyer la réponse*).
In the adventure of icaRius, you will sometimes be asked to answer quizzes to help the character continue his quest. Here is a very first one, just to familiarize yourself with the format of the exercise. So it's up to you to respond directly by checking your choice and then pressing the button called `Submit answer`.

> À ce sujet, vous croiserez de temps à autre quelques mots étranges issus de la langue de Shakespeare. Le langage runique emprunte en effet quelques termes à nos voisins *grands brittons*... Mais le grimoire vous proposera toujours la traduction dans la langue - infiniment plus mélodieuse - de Molière...

Author: Je me suis permis de supprimer ce paragraphe :wink:

Reviewer: Complètement d'accord - ah, ces frogs et leur humour anti-british :-p

    answer("a free software is a wild software, it cannot be found at home"),
    answer("a free software is a software that freely authorizes the use, study, modification and redistribution", correct = TRUE),

>     answer("free software is software that has been seen flying")

Author: J'ai de sérieux doutes sur le fait que cette chanson soit connue des anglophones. La traduction littérale ne veut certainement rien dire.

Reviewer: En effet, il faut trouver autre chose... quitte à ce qu'il n'y ait aucun lien.
```suggestion
    answer("a free software is a software working only with escape key")
```

```

```{r ok-chapitre1-etape1,exercise=TRUE}
# Création des variables nb_coeur et max_nb_coeur
# Create the object n_lives and max_n_lives

# Affichage de leur valeur
# Display their values

# Fin de l'exercice
# End of the exercise
``` 
  

```{r ok-chapitre1-etape1-solution}
# Création des variables nb_coeur et max_nb_coeur
nb_coeur<-3
max_nb_coeur<-2*nb_coeur
# Affichage de leur valeur
nb_coeur
max_nb_coeur
# Fin de l'exercice
# Create the object n_lives and max_n_lives
n_lives<-3
max_n_lives<-2*n_lives
# Display their values
n_lives
max_n_lives
# End of the exercise
```   

<span style="color:#18AC3E;font-size:16px">**Des objets préchargés**</span>
<span style="color:#18AC3E;font-size:16px">**Preloaded objects**</span>

Pour vous aider dans votre quête, le langage des Runes comporte déjà quelques "objets" déjà rangés à l'avance. Par exemple, la valeur de pi est stockée dans l'objet `pi`. 
To help you in your quest, the language of the Runes already has a few “items” already stored in advance. For example, the value of pi is stored in the object `pi`. 

```{r, collapse = TRUE}
# Afficher la valeur de pi dans R
# Display the value of pi in R
pi
```

> Parlons un instant des décimales. Vous avez l'habitude d'utiliser une virgule pour indiquer les décimales dans un nombre, non ? Comme 3,1415 pour pi ? Certes, mais les pays anglophones ont choisi le point comme séparateur décimal, et c'est cette convention que l'on retrouve dans les langages comme R. C'est pour cela que R affiche `3.14` et non `3,14`. Il vous faudra vous plier à cette règle dans votre apprentissage du langage des Runes sous peine de vous faire rouspeter par la console.

Author: Là encore, ce paragraphe semble sans intérêt dans la version anglaise.

Reviewer: C'est juste. Comme quoi, la vie est plus simple pour l'apprenti anglosaxon...
